frontend/src/metabase/embedding-sdk/components/SimpleDataPicker/SimpleDataPicker.tsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import EmptyState from "metabase/components/EmptyState";
+import { CONTAINER_WIDTH } from "metabase/query_builder/components/DataSelector/constants";
+import { Flex, Icon, NavLink, Paper, ScrollArea, TextInput } from "metabase/ui";
+import type { TableId } from "metabase-types/api";
+
+interface SimpleDataPickerProps {
+  selectedEntity?: TableId;
+  options: Options[];
+  onClick: (option: any) => void;
+}
+
+interface Options {
+  id: number;
+  display_name: string;
+}
+
+const TEN_OPTIONS_HEIGHT = 10 * 33;
+
+export function SimpleDataPicker({
+  selectedEntity,
+  options,
+  onClick,
+}: SimpleDataPickerProps) {
+  const shouldShowSearchBar = options.length > 10;
+  const [searchText, setSearchText] = useState("");
+  function filterSearch(option: Options): boolean {
+    if (searchText) {
+      return normalizeString(option.display_name).includes(
+        normalizeString(searchText),
+      );
+    }
+
+    return true;
+  }
+
+  if (options.length === 0) {
+    return (
+      <Paper w={CONTAINER_WIDTH} p="80px 60px">
+        <EmptyState
+          message={t`To pick some data, you'll need to add some first`}
+          icon="database"
+        />
+      </Paper>
+    );
+  }
+
+  const displayOptions = options.filter(filterSearch);
+  return (
+    <Paper
+      component={Flex}
+      w={CONTAINER_WIDTH}
+      p="sm"
+      mih="200px"
```

Comment:
Add a min-height to support an empty state.
![Screenshot 2025-02-04 at 7 48 38 PM](https://github.com/user-attachments/assets/fb9d7ac3-7b51-4c5e-95d3-71059acf76c5)

This also works when there are really few data sources, we won't get a popover that's too short anymore.

![Screenshot 2025-02-04 at 7 48 41 PM](https://github.com/user-attachments/assets/236f9c83-a672-4baa-a9c4-632783d785c4)